uint8 camera_mode

bool zoom_in
bool zoom_out
bool full_zoom_in
bool full_zoom_out
bool shoot